third-party/tbb/pic_compile.patch

```diff
@@ -0,0 +1,13 @@
+diff --git a/third-party/tbb/src/build/linux.gcc.inc b/third-party/tbb/src/build/linux.gcc.inc
```

Comment:
What does this file do?

Reply:
This is a patch to enable `-fPIC` when compiling with GCC. Without it, we would get an error when compiling TBB.

All the files in the directory have `PIC_KEY` set and it's the only one that comments it.  TBB uses those `<os>-<compiler>.inc` files to set compilation options.